### ISSUE TITLE
[Toolbar] Toolbar elements should have `aria-disabled` instead of `disabled` attribute when `disabled` is set on the toolbar

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2972]
### Provide a general summary of the issue here

When the toolbar is set as `disabled`, the elements inside should have `aria-disabled` set instead of `disabled`. 

Please make sure that keyboard navigation is working fine (elements with `aria-disabled` should be focusable).

Please improve all examples with `disabled` buttons / menu-buttons as well.

\
Please check this comment from Kasia as well: https://github.com/sl-design-system/components/issues/2748#issuecomment-3804545101

### 👤 Your name

SLDS Team

### 🧢 Your Product/Team

SLDS
